lib/string_decoder.js

```diff
@@ -42,6 +42,7 @@ const {
 } = internalBinding('string_decoder');
 const internalUtil = require('internal/util');
 const {
+  ERR_ILLEGAL_CONSTRUCTOR,
```

Comment:
Wouldn't `ERR_INVALID_THIS` be more appropriate here?

Reply:
Yep, will fix the error code once the benchmark run completes.

Reply:
Done, PTAL